src/cogent3/core/sequence.py

```diff
@@ -736,6 +737,9 @@ def get_color_scheme(self, colors):  # alias to support US spelling
     def copy_annotations(self, other):
         self.annotations = other.annotations[:]
 
+    def copy(self):
+        return copy.deepcopy(self)
```

Comment:
suggest using `self.__class__` passing all required attributes, but you should do deepcopy() on the `info` attribute (but check to see if it's already copied in the constructor).